packages/split-view/README.md

```diff
@@ -0,0 +1,139 @@
+## Description
+
+### Usage
+
+[![See it on NPM!](https://img.shields.io/npm/v/@spectrum-web-components/split-view?style=for-the-badge)](https://www.npmjs.com/package/@spectrum-web-components/split-view)
+[![How big is this package in your project?](https://img.shields.io/bundlephobia/minzip/@spectrum-web-components/split-view?style=for-the-badge)](https://bundlephobia.com/result?p=@spectrum-web-components/split-view)
+
+```
+yarn add @spectrum-web-components/split-view
+```
+
+Import the side effectful registration of `<sp-split-view>` via:
+
+```
+import '@spectrum-web-components/split-view/sp-split-view.js';
+```
+
+When looking to leverage the `SplitView` base class as a type and/or for extension purposes, do so via:
+
+```
+import { SplitView } from '@spectrum-web-components/split-view';
+```
+
+### Variants
+
+#### Horizontal
+
+```html
+<sp-split-view>
+    <div>Left panel</div>
+    <div>Right panel</div>
+</sp-split-view>
+```
+
+#### Horizontal Resizable
+
+```html
+<sp-split-view
+    resizable
+    primary-min="50"
+    primary-max="Infinity"
```

Comment:
Agree, will use 3840 (4k) as default_max_size.

Reply:
Probably fine either way.

My point here is that a max that could be experienced on the page, like `primary-max="400"`, would allow the user to see that it actually does something, e.g. prevents the main pane from getting larger than 400px.

Reply:
Hm, can you please explain where it wasn't happen for you? Tried to find why it may not working as expected with -max values. It worked for me so far.